sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/InstancesSamples.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using static Azure.IoT.TimeSeriesInsights.Samples.SampleLogger;
+
+namespace Azure.IoT.TimeSeriesInsights.Samples
+{
+    internal class InstancesSamples
+    {
+        /// <summary>
+        /// </summary>
+        public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
+        {
+            PrintHeader("TIME SERIES INSIGHTS INSTANCES SAMPLE");
+
+            // Figure out how many keys make up the Time Series Id
+            TimeSeriesModelSettings modelSettings = await client.ModelSettings.GetAsync().ConfigureAwait(false);
+
+            TimeSeriesId instanceId = modelSettings.TimeSeriesIdProperties.Count switch
+            {
+                1 => new TimeSeriesId("key1"),
+                2 => new TimeSeriesId("key1", "key2"),
+                3 => new TimeSeriesId("key1", "key2", "key3"),
+                _ => throw new Exception($"Invalid number of Time Series Insights Id properties."),
+            };
+
+            #region Snippet:TimeSeriesInsightsSampleCreateInstance
+
+            // Create a Time Series Instance object with the default Time Series Insights type Id.
+            // The default type Id can be obtained programmatically by using the ModelSettings client.
+            var instance = new TimeSeriesInstance(instanceId, "1be09af9-f089-4d6b-9f0b-48018b5f7393")
```

Comment:
I also thought about this when I implemented this. Initially I thought it would be better to hardcode the default type guid. But looking at it again, it's better to replace it with the Id retrieved from ModelSettings. I also think the comment above the line should help explain how we retrieved the value. 

Will change it.